astropy/tests/helper.py

```diff
@@ -29,45 +29,6 @@
 IS_CRON = os.environ.get("IS_CRON", "false") == "true"
 
 
-def _save_coverage(cov, result, rootdir, testing_path):
```

Comment:
This is only used in `command.py` that is deleted in this PR, so we can delete this too.